static/js/util/profile_edit.js

```diff
@@ -71,7 +73,68 @@ export function boundDateField(key, label, error) {
     <span className="validation-error-text">{error}</span>
   </div>;
 }
+export function boundEducationMonthField(keySet, label) {
+  const {updateProfile, profile} = this.props;
+  let onChange = month => {
+    let clone = Object.assign({}, profile);
```

Comment:
Can you change this to use `_.cloneDeep`? We will need to do a deep clone here since `_.set` can modify any object within the clone, not just the root object.